spec/osc-server-spec.js

```diff
@@ -0,0 +1,33 @@
+const OscServer = require('../lib/osc-server')
+const osc = require('osc-min')
+const dgram = require('dgram');
+
+describe('OscServer', () => {
```

Comment:
if this is really the case [here](https://github.com/tidalcycles/pulsar-tidalcycles/pull/218/files#r2213353643), then we should add the use case for simulating multiple senders here somehow.